scorescreen.py

```diff
@@ -124,11 +126,4 @@ def run(self, gameDisplay, scores):
             if crashed == True:                       # Game crash or Close check
                 pygame.quit()
                 sys.exit()
-
-        # Just a window exception check condition
-        event1 = pygame.event.get()
-        if event1.type == pygame.QUIT:
-            crashed = True
-        if crashed == True:
-            pygame.quit()
-            sys.exit()
+        
```

Comment:
Removed the part where events were dequeued for the second time.

Reply:
Again.